src/lib/ngsiNotify/senderThread.cpp

```diff
@@ -30,6 +30,18 @@
 #include "ngsiNotify/senderThread.h"
 
 
+// FIXME P11 #1669: remove this stub when the actual sendNotifyContextRequestWs() gets developed a the end. @fortizc please take care of this
+// in the ws library
+int sendNotifyContextRequestWs(const std::string& subId, const std::map<std::string, std::string>& headers, const std::string& data)
```

Comment:
At the begining I proposed (at #1669) to use `bool` as return value for this function. However, in order to align with httpRequestSent() signature used at startSenderThread() function, I think it should be better to use `int` the the usual semantics: 0 -> everythin ok, not 0 -> error.

(I have edited signature at https://github.com/telefonicaid/fiware-orion/issues/1669#issuecomment-199907859 has been changed `void` -> `int`)